docs/user/resources/05-metricpipeline.md

```diff
@@ -51,10 +51,10 @@ For details, see the [MetricPipeline specification file](https://github.com/kyma
 | ---- | ----------- | ---- |
 | **input**  | object | Configures different inputs to send additional metrics to the metric gateway. |
 | **input.&#x200b;application**  | object | Configures application related scraping. |
+| **input.&#x200b;application.&#x200b;prometheus**  | object | Configures Prometheus scraping. |
+| **input.&#x200b;application.&#x200b;prometheus.&#x200b;enabled**  | boolean | If enabled, Pods marked with prometheus.io/scrape=true annotation will be scraped. |
```

Comment:
```suggestion
| **input.&#x200b;application.&#x200b;prometheus.&#x200b;enabled**  | boolean | If enabled, Pods marked with `prometheus.io/scrape=true` annotation will be scraped. |
```

Reply:
Unfortunately, adding ticks, quotes, etc. is a bit problematic. The README is generated from a CRD definition (YAML file), which, in turn, is generated from a Go struct.

Reply:
Fixed